tests/build_local_test_elfs.sh

```diff
@@ -1,8 +1,8 @@
 #!/bin/bash
 
 # FILL THESE WITH YOUR OWN SDKs PATHS and APP-ETHEREUM's ROOT
-NANOS_SDK=$NANOS_SDK
```

Comment:
This script was [re-written in boilerplate repo](https://github.com/LedgerHQ/app-plugin-boilerplate/blob/master/tests/build_local_test_elfs.sh), some code may be useful (?)

Reply:
The new shell script from boilerplate will be added in a future PR.